src/components/Button.js

```diff
@@ -3,18 +3,26 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const { name, color, twice } = props;
```

Comment:
- I highly recommend that you use the same prop names as those in the project specifications, replace `twice` with `wide` [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorAndWideProps.png)